topics/ecology/tutorials/obisindicators/tutorial.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: tutorial_hands_on
+
+title: Obis marine indicators
+zenodo_link: ''
+questions:
+objectives:
+- Calculating and vizualizing marine biodiversity indicators
+time_estimation: 1H
+key_points:
+- Marine data
+- Diversity indicators
+contributors:
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Reply:
(8f this is for gallantries please indicate accordingly)